addons/rook/template/base/install.sh

```diff
@@ -339,7 +339,57 @@ function rook_cluster_deploy_upgrade() {
 
     rook_cluster_deploy_upgrade_flexvolumes_to_csi
 
-    logSuccess "Rook-ceph cluster upgraded"
+    verify_rook_updated_cluster
```

Comment:
Nit but can you move this log line back to the calling function?

```suggestion
    logStep "Checking if the Rook-Ceph cluster upgrade completed successfully"

    verify_rook_updated_cluster

    logSuccess "Rook-Ceph cluster upgraded successfully"
```